### ISSUE TITLE
Drag and drop file manager

### ISSUE THREAD

**Issue by leofeyer** [contao/core-bundle#1403]
I have noticed two issues with the new drag and drop implementation for the file manager.

1. Clicking an element without draging it shows the clone:

<img width="551" alt="" src="https://user-images.githubusercontent.com/1192057/36785581-9891cc2a-1c83-11e8-9269-2096467ce4a3.png">

2. All other draggable elements in the back end use a drag handle. For reasons of consistency, I think we should add a drag handle in the file manager, too.

**Comment by Toflar** [comment]
Can we try highlighting the file name with a different color like on OSX? It really helps UX a lot. I would love to get some screenshots so we can decide which one looks best 😎

**Comment by leofeyer** [comment]
Also see https://github.com/contao/core-bundle/pull/1394#issuecomment-377721950

> @ausi You should ignore right (context menu) clicks with an check like:
> ``
> if (event.rightClick || event.control) {
> }
> ``

**Comment by ausi** [comment]
Yeah, drag handle seams reasonable. Should I add it as the last action button like it is in the content elements view?

**Comment by leofeyer** [comment]
Yes please.

**Comment by leofeyer** [comment]
I have found another small issue: the clone is also created upon selecting a row.

https://www.dropbox.com/s/f3ydj23i2r61907/1403.mp4?dl=0

I guess this will resolve itself when we switch to using a drag handle?

**Comment by ausi** [comment]
Yes, I think so too.
